lib/processor.js

```diff
@@ -1,6 +1,11 @@
 /**
  * @fileoverview Processes Markdown files for consumption by ESLint.
  * @author Brandon Mills
+ *
+ * @typedef {import('unist').Node} ASTNode
+ * @typedef {ASTNode & {children: ASTNode[]}} ASTParentNode
+ * @typedef {import('eslint').Linter.LintMessage} Message
```

Comment:
I have no idea how JSDoc imports work, but instead of `@types/eslint`, does it support importing the `LintMessage` `@typedef` provided by ESLint in [`lib/shared/types.js`](https://github.com/eslint/eslint/blob/cd2e584aa5a87c425177fb84a9cd6b05624fec43/lib/shared/types.js#L84-L96)? We don't strip comments from our builds, so those comments are all present in `node_modules` installs.